From 87ea3e07b939fecc0d2e1d86d8320e7113ab3fa0 Mon Sep 17 00:00:00 2001
From: Luigi Toscano <luigi.toscano@tiscali.it>
Date: Wed, 16 Jun 2021 01:10:10 +0200
Subject: [PATCH 1/3] Minimal changes to use libera.chat

Change the reference to libera in non-translatable strings first,
so that this patch can be easily cherry-picked and backported.

CCBUG: 437589
---
 README                     | 2 +-
 src/config/preferences.cpp | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/README b/README
index 1009c751..02b51c49 100644
--- a/README
+++ b/README
@@ -1,4 +1,4 @@
 Konversation website: https://konversation.kde.org/
 Konversation wiki: https://userbase.kde.org/Konversation
 Bug tracker: https://bugs.kde.org/
-IRC channel: #konversation on chat.freenode.net
+IRC channel: #konversation on irc.libera.chat
diff --git a/src/config/preferences.cpp b/src/config/preferences.cpp
index 48d94673..8a28caa7 100644
--- a/src/config/preferences.cpp
+++ b/src/config/preferences.cpp
@@ -57,9 +57,9 @@ Preferences::Preferences()
     mIdentity->setNicknameList(nickList);
 
     Konversation::ServerGroupSettingsPtr serverGroup(new Konversation::ServerGroupSettings);
-    serverGroup->setName(QStringLiteral("freenode"));
+    serverGroup->setName(QStringLiteral("libera"));
     Konversation::ServerSettings server;
-    server.setHost(QStringLiteral("chat.freenode.net"));
+    server.setHost(QStringLiteral("irc.libera.chat"));
     server.setPort(6697);
     server.setSSLEnabled(true);
     serverGroup->addServer(server);
-- 
2.32.0

